gnovm/pkg/gnolang/op_binary.go

```diff
@@ -71,6 +71,17 @@ func (m *Machine) doOpLand() {
 	lv.SetBool(lv.GetBool() && rv.GetBool())
 }
 
+// setMaybeUntypedBool sets lv to b; it will be untyped if both lv and rv are
+// untyped; otherwise, it will be a typed bool.
+func setMaybeUntypedBool(lv, rv *TypedValue, b bool) {
```

Comment:
the change in this file should be unnecessary anymore, since the it's fixed, in another way. see: https://github.com/gnolang/gno/pull/3298

Reply:
yes, reverted!